esi_leap/tests/api/controllers/v1/test_lease.py

```diff
@@ -729,7 +729,7 @@ def test_lease_get_all_status(self):
 
         expected_filters = {
             'project_or_owner_id': 'adminid',
-            'status': ['created', 'active', 'error']
+            'status': ['created', 'active', 'error', 'wait']
```

Comment:
Not really your fault here since this problem was pre-existing, but these should probably be references to statuses.CREATED/ACTIVE/etc. rather than their string values.

Reply:
Yeah, you're completely right - fixed!